docs/src/tutorial/getting-started.md

```diff
@@ -1 +1,187 @@
+namespace:
+
+```
+sudo k8s kubectl get pods -n kube-system
+```
+
+You will observe three pods running:
+- **coredns**: Provides DNS resolution services.
+- **network-operator**: Manages the lifecycle of the networking solution.
+- **networking agent**: Facilitates network management.
+
+Confirm that Canonical Kubernetes has transitioned to the `k8s is ready` state by running:
+
+```
+sudo k8s status
+```
+
+Note: To disable a component execute `sudo k8s disable <component>`
+
+### 5. Access Kubernetes
+The standard tool for deploying and managing workloads on Kuberenetes
+is [kubectl](https://kubernetes.io/docs/reference/kubectl/). 
+For convenience, Canonical Kubernetes bundles a version of 
+kubectl for you to use with no extra setup or configuration. 
+For example, to view your node you can run the command:
+
+```
+sudo k8s kubectl get nodes
+```
+
+…or to see the running services:
+
+```
+sudo k8s kubectl get services
+```
+
+### 6. Deploy an app
+
+Kubernetes is meant for deploying apps and services. 
+You can use the `kubectl`
+command to do that as with any Kubernetes. 
+
+Let's deploy a demo NGINX server:
+
+```
+sudo k8s kubectl create deployment nginx --image=nginx
+```
+This command launches a [pod](https://kubernetes.io/docs/concepts/workloads/pods/),
+the smallest deployable unit in Kubernetes,
+running the nginx application within a container.
+
+You can check the status of your pods by running:
+
+```
+sudo k8s kubectl get pods
+```
+
+This command shows all pods in the default namespace. 
+It may take a moment for the pod to be ready and running.
+
+### 7. Remove an app
+To remove the NGINX workload, execute the following command:
+```
+sudo k8s kubectl delete deployment nginx --image=nginx
+
+```
+
+To verify that the pod has been removed, you can check the status of pods by running:
+
+```
+sudo k8s kubectl get pods
+```
+### 8. Enable Components (Storage)
+In scenarios where you need to preserve application data beyond the 
+lifecycle of the pod, Kubernetes provides persistent volumes.
+
+With Canonical Kubernetes, you can enable storage to configure 
+your storage solutions:
+
+```
+sudo k8s enable storage
+```
+
+To verify that the storage component is enabled, execute:
+```
+sudo k8s status
+```
+You should see `storage enabled` in the command output.
+
+Let's create a persistent volume:
+```
+kubectl apply -f https://k8s.io/examples/pods/storage/pv-volume.yaml
+```
+This command deploys a pod based on the YAML configuration of a 
+persistent volume with a capacity of 10G.
+
+To confirm that the persistent volume is up and running:
+```
+sudo k8s kubectl get pv task-pv-volume
+```
+
+### 9. Disable Components (Storage)
+Begin by removing the persistent volume:
+```
+sudo k8s kubectl delete pv task-pv-volume
+```
+
+Next, disable the storage component:
+
+```
+sudo k8s disable storage
+```
+
+Note: To disable any component, execute  `sudo k8s disable <component>`.
+
```

Comment:
I would also add the optional step of turning off k8s and also uninstalling the snap, just for completeness!